File: EmbyStat.Web/ClientApp/src/app/shared/components/language/service/language.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { Observable } from 'rxjs';

import { Language } from '../models/language';

@Injectable()
export class LanguageService {
  private readonly getLangaugesUrl: string = '/language/getlist';

  constructor(private http: HttpClient) { }

  getLanguages(): Observable<Language[]> {
    return this.http.get<Language[]>('/api' + this.getLangaugesUrl);
  }
}

File: EmbyStat.Core/Languages/LanguageService.cs
using EmbyStat.Common.Models.Entities;
using EmbyStat.Core.Languages.Interfaces;

namespace EmbyStat.Core.Languages;

public class LanguageService : ILanguageService
{
    private readonly ILanguageRepository _languageRepository;

    public LanguageService(ILanguageRepository languageRepository)
    {
        _languageRepository = languageRepository;
    }

    public Task<List<Language>> GetLanguages()
    {
        return _languageRepository.GetLanguages();
    }
}